core/src/main/java/com/graphhopper/routing/weighting/Weighting.java

```diff
@@ -37,6 +39,8 @@
      */
     double getMinWeight(double distance);
 
+    double calcEdgeWeight(EdgeIteratorState edgeState, boolean reverse);
```

Comment:
Can you add javadocs? And highlight the difference to calcWeight?

Reply:
Sure, good point!